pkg/services/rocketchat_test.go

```diff
@@ -0,0 +1,45 @@
+package services
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Would you sort the order as same other package, please?

Reply:
Fixed